server.go

```diff
@@ -130,6 +130,20 @@ func runMain() error {
 			err)
 	}
 
+	coldwalletResponse, err := controller.Cfg.StakepooldServers.GetColdWalletExtPub()
+	if err != nil {
+		log.Infof("RPC GetColdWalletExtPub failed: %v", err)
+		return fmt.Errorf("GetColdWalletExtPub: RPC server error %v", err)
+	}
+
+	// Check that coldwalletextpub is the same in stakepoold.conf and dcrstakepool.conf files
+	if cfg.ColdWalletExtPub != coldwalletResponse.ColdWalletExtPub {
+		str := "%s: coldwalletextpub is not the same in stakepoold.conf and dcrstakepool.conf files"
+		err := fmt.Errorf(str, "GetColdWalletExtPub")
```

Comment:
This isn't a call to `GetColdWalletExtPub`, so the error message should not include this. Should simply return an error as is done below:
`return fmt.Errof("coldwalletextpub is not the same in stakepoold.conf and dcrstakepool.conf files")`

Reply:
OK